docs/about/hyp3_basic.md

```diff
@@ -0,0 +1,23 @@
+# HyP3 Basic
+
+ASF released HyP3 to the public in 2020, allowing all users to request On Demand products at no cost. 
+This service, now called HyP3 Basic, is funded by NASA, and hosted in 
+NASA's [Earthdata Cloud (EDC)](https://www.earthdata.nasa.gov/about/earthdata-cloud-evolution "https://www.earthdata.nasa.gov/about/earthdata-cloud-evolution" ){target=_blank}. 
+
+Anyone can use HyP3 Basic. All you need are 
+[Earthdata Login credentials](../using/authentication.md#earthdata-login-edl "Jump to EDL Documentation"). 
+
+Each user receives 10,000 credits each month to use for processing any On Demand products. Different HyP3 job types 
+consume different credit amounts, as defined in the 
+[Credit Cost Table](../using/credits.md#credit-cost-table "Jump to the Credit Cost Table on the Credits Documentation page"). 
+If you need to process more data than is possible with this credit allotment, you can purchase additional credits using 
+[HyP3+](hyp3_plus.md "Jump to HyP3+ Documentation"). 
+
+Output products are hosted in an S3 bucket in EDC, and are retained for 14 days before being deleted. 
```

Comment:
```suggestion
Output products are distributed via Amazon CloudFront, and are retained for 14 days before being deleted. 
```

We don't allow user access to the S3 buckets in EDC, so I think it's better to not mention S3 at all here.